backoffice/src/resources/Pro/Offerers/OffererDetail.tsx

```diff
@@ -97,7 +100,14 @@ export const OffererDetail = () => {
     'offerer',
     { id: id },
     // redirect to the list if the book is not found
-    { onError: () => redirect('/pro/search') }
+    {
+      onError: () => {
```

Comment:
Pas besoin d'arrow function 
```suggestion
      onError() {
```